src/vs/workbench/contrib/files/browser/fileActions.ts

```diff
@@ -143,6 +143,27 @@ export class NewFolderAction extends Action {
 	}
 }
 
+/* Create new file from anywhere: Open untitled */
+// {{SQL CARBON EDIT}}
+export class GlobalNewUntitledPlainFileAction extends Action {
+	// {{SQL CARBON EDIT}} - Use different command name to reserve original name for SQL files
```

Comment:
{{SQL CARBON EDIT}} tag is probably not needed here since the method is already marked by it.

Reply:
Yeah, just trying to make it explicit what's changed.  This can help when looking at merge conflicts and try to reconstruct the original patch.